tracer/src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs

```diff
@@ -46,5 +48,71 @@ public async Task<IApiResponse> PostAsync(ArraySegment<byte> bytes, string conte
                 return new ApiWebResponse((HttpWebResponse)exception.Response);
             }
         }
+
+        public async Task<IApiResponse> PostAsync(params MultipartFormItem[] items)
+        {
+            if (items is null)
+            {
+                ThrowHelper.ThrowArgumentNullException(nameof(items));
+            }
+
+            Log.Debug<int>("Sending multipart form request with {Count} items.", items.Length);
+
+            var boundary = "---------------------------" + DateTime.Now.Ticks.ToString("x");
```

Comment:
Also, just wondering out loud, do we _have_ to use a new value for boundary each time here? If we _actually_ needed unique values then you presumably wouldn't be using `DateTime.Now.Ticks`...

If that's the case, could we cache the `nowTicks` the first time? Then we could potentially cache some of the the byte arrays and avoid some of the `Encoding.ASCII.GetBytes` below?

Reply:
I made a change to use a constant in [84ecdc7](https://github.com/DataDog/dd-trace-dotnet/pull/2962/commits/84ecdc7a6638c4593e83777456e9370644539de4)